luigi/scheduler.py

```diff
@@ -366,13 +379,31 @@ def get_active_tasks(self, status=None):
             for task in six.itervalues(self._tasks):
                 yield task
 
+    def get_batch_running_tasks(self, batch_id):
+        if batch_id is None:
```

Comment:
I'll just have to add the same conditional around each call to this function. I'll do that if you prefer.